src/main/java/core/basesyntax/service/strategy/EvaluateResult.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.service.strategy;
+
+import core.basesyntax.service.model.FruitTransaction;
+import java.util.List;
+
+public interface EvaluateResult {
```

Comment:
let`s move it to service package and rename it to FruitShopService, becouse it is not strategy